.github/workflows/maven-verify.yml

```diff
@@ -24,7 +24,10 @@ on:
 jobs:
   build:
     name: Verify
-    uses: apache/maven-gh-actions-shared/.github/workflows/maven-verify.yml@v2
+    uses: apache/maven-gh-actions-shared/.github/workflows/maven-verify.yml@v3
     with:
-      jdk-distribution-matrix: '[ "temurin", "zulu", "microsoft", "liberica", "adopt-openj9" ]'
-      matrix-exclude: '[{ "jdk": "8", "distribution": "microsoft"}]'
+      maven-args: "-Dinvoker.parallelThreads=2"
```

Comment:
Why not in project configuration?
I like values as 1C, 2C, ...

Reply:
because need some good environment and might not suit everyone.

Reply:
btw doesn't work (e.g concurrent access failure https://github.com/apache/maven-compiler-plugin/runs/7444447535?check_suite_focus=true)

Reply:
looks like problem in Maven itself

```
Caused by: java.io.FileNotFoundException: .../local-repo/org/sonatype/plexus/plexus-build-api/0.0.4/plexus-build-api-0.0.4.pom.part (No such file or directory)
``

Reply:
yup concurrent download of artifact